### ISSUE TITLE
Pytest error with test_env

### ISSUE THREAD

**Issue by MarshallYan** [choderalab/modelforge#175]
I got these errors when I installed the test environment from 'test_env.yaml'. It might be an error with 'pint' version since 'pint.Measurement' does exist. However, openff-units does not specify a pint version, nor does modelforge. Has anyone seen this before? Thanks.

```
ImportError while loading conftest '/home/shuaiy/workdir/modelforge/modelforge/tests/conftest.py'.
conftest.py:3: in <module>
    from modelforge.dataset import DataModule
../dataset/__init__.py:2: in <module>
    from .qm9 import QM9Dataset
../dataset/qm9.py:3: in <module>
    from .dataset import HDF5Dataset
../dataset/dataset.py:9: in <module>
    from openff.units import Quantity
../../../../miniforge3/envs/test/lib/python3.11/site-packages/openff/units/__init__.py:4: in <module>
    from openff.units.units import DEFAULT_UNIT_REGISTRY, Measurement, Quantity, Unit
../../../../miniforge3/envs/test/lib/python3.11/site-packages/openff/units/units.py:11: in <module>
    from pint.measurement import _Measurement
E   ModuleNotFoundError: No module named 'pint.measurement'
```

P.S. I have a Fedora machine running on AMD CPU and AMD GPU platform.

**Comment by chrisiacovella** [comment]
Can you post the versions of pint and opened- units installed (use `conda list` to get the versions in your environment)

**Comment by MarshallYan** [comment]
> Can you post the versions of pint and opened- units installed (use `conda list` to get the versions in your environment)

```
# packages in environment at /home/shuaiy/miniforge3/envs/test:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                  2_kmp_llvm    conda-forge
absl-py                   2.1.0              pyhd8ed1ab_0    conda-forge
aiohttp                   3.9.5           py311h459d7ec_0    conda-forge
aiohttp-cors              0.7.0                      py_0    conda-forge
aiosignal                 1.3.1              pyhd8ed1ab_0    conda-forge
alabaster                 0.7.16             pyhd8ed1ab_0    conda-forge
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
aom                       3.9.1                hac33072_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
apsw                      3.46.0.0        py311h3ea06b8_0    conda-forge
ase                       3.23.0             pyhd8ed1ab_0    conda-forge
astunparse                1.6.3                    pypi_0    pypi
async-timeout             4.0.3              pyhd8ed1ab_0    conda-forge
atk-1.0                   2.38.0               h04ea711_2    conda-forge
attrs                     23.2.0             pyh71513ae_0    conda-forge
aws-c-auth                0.7.22               hf36ad8f_6    conda-forge
aws-c-cal                 0.6.15               h816f305_1    conda-forge
aws-c-common              0.9.23               h4ab18f5_0    conda-forge
aws-c-compression         0.2.18               he027950_7    conda-forge
aws-c-event-stream        0.4.2               hb72ac1a_14    conda-forge
aws-c-http                0.8.2                h75ac8c9_3    conda-forge
aws-c-io                  0.14.9               hd3d3696_3    conda-forge
aws-c-mqtt                0.10.4               hb0abfc5_7    conda-forge
aws-c-s3                  0.5.10               h44b787d_4    conda-forge
aws-c-sdkutils            0.1.16               he027950_3    conda-forge
aws-checksums             0.1.18               he027950_7    conda-forge
aws-crt-cpp               0.26.12              he940a02_1    conda-forge
aws-sdk-cpp               1.11.329             h0f5bab0_6    conda-forge
azure-core-cpp            1.12.0               h830ed8b_0    conda-forge
azure-identity-cpp        1.8.0                hdb0d106_1    conda-forge
azure-storage-blobs-cpp   12.11.0              ha67cba7_1    conda-forge
azure-storage-common-cpp  12.6.0               he3f277c_1    conda-forge
azure-storage-files-datalake-cpp 12.10.0              h29b5301_1    conda-forge
babel                     2.14.0             pyhd8ed1ab_0    conda-forge
blinker                   1.8.2              pyhd8ed1ab_0    conda-forge
blosc                     1.21.6               hef167b5_0    conda-forge
brotli                    1.1.0                hd590300_1    conda-forge
brotli-bin                1.1.0                hd590300_1    conda-forge
brotli-python             1.1.0           py311hb755f60_1    conda-forge
brunsli                   0.1                  h9c3ff4c_0    conda-forge
bzip2                     1.0.8                hd590300_5    conda-forge
c-ares                    1.28.1               hd590300_0    conda-forge
c-blosc2                  2.15.0               h6d6b9e4_1    conda-forge
ca-certificates           2024.6.2             hbcca054_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
cachetools                5.3.3              pyhd8ed1ab_0    conda-forge
cairo                     1.18.0               hbb29018_2    conda-forge
captum                    0.6.0              pyhd8ed1ab_0    conda-forge
certifi                   2024.6.2           pyhd8ed1ab_0    conda-forge
cffi                      1.16.0          py311hb3a22ac_0    conda-forge
chardet                   5.2.0           py311h38be061_1    conda-forge
charls                    2.4.2                h59595ed_0    conda-forge
charset-normalizer        3.3.2              pyhd8ed1ab_0    conda-forge
chex                      0.1.86                   pypi_0    pypi
click                     8.1.7           unix_pyh707e725_0    conda-forge
codecov                   2.1.13             pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
colorful                  0.5.6              pyhd8ed1ab_0    conda-forge
colorlog                  6.8.2                    pypi_0    pypi
contourpy                 1.2.1           py311h9547e67_0    conda-forge
coverage                  7.5.4           py311h331c9d8_0    conda-forge
cryptography              42.0.8          py311h4a61cc7_0    conda-forge
cycler                    0.12.1             pyhd8ed1ab_0    conda-forge
dav1d                     1.2.1                hd590300_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
dirsync                   2.2.5                    pypi_0    pypi
distlib                   0.3.8              pyhd8ed1ab_0    conda-forge
docutils                  0.20.1          py311h38be061_3    conda-forge
elfutils                  0.191                h924a536_0    conda-forge
etils                     1.9.2                    pypi_0    pypi
exceptiongroup            1.2.0              pyhd8ed1ab_2    conda-forge
expat                     2.6.2                h59595ed_0    conda-forge
fasteners                 0.19                     pypi_0    pypi
filelock                  3.15.4             pyhd8ed1ab_0    conda-forge
flask                     3.0.3              pyhd8ed1ab_0    conda-forge
flatbuffers               24.3.25                  pypi_0    pypi
flax                      0.8.5                    pypi_0    pypi
flexcache                 0.3                pyhd8ed1ab_0    conda-forge
flexparser                0.3.1              pyhd8ed1ab_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 h77eed37_2    conda-forge
fontconfig                2.14.2               h14ed4e7_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.53.0          py311h331c9d8_0    conda-forge
freetype                  2.12.1               h267a509_2    conda-forge
freetype-py               2.3.0              pyhd8ed1ab_0    conda-forge
fribidi                   1.0.10               h36c2ea0_0    conda-forge
frozenlist                1.4.1           py311h459d7ec_0    conda-forge
fsspec                    2024.6.1           pyhff2d567_0    conda-forge
gast                      0.6.0                    pypi_0    pypi
gdk-pixbuf                2.42.12              hb9ae30d_0    conda-forge
gettext                   0.22.5               h59595ed_2    conda-forge
gettext-tools             0.22.5               h59595ed_2    conda-forge
gflags                    2.2.2             he1b5a44_1004    conda-forge
giflib                    5.2.2                hd590300_0    conda-forge
glog                      0.7.1                hbabe93e_0    conda-forge
gmp                       6.3.0                hac33072_2    conda-forge
gmpy2                     2.1.5           py311hc4f1f91_1    conda-forge
gnutls                    3.7.9                hb077bed_0    conda-forge
google-api-core           2.19.1             pyhd8ed1ab_0    conda-forge
google-auth               2.30.0             pyhff2d567_0    conda-forge
google-pasta              0.2.0                    pypi_0    pypi
googleapis-common-protos  1.63.2             pyhd8ed1ab_0    conda-forge
graphite2                 1.3.13            h59595ed_1003    conda-forge
graphviz                  11.0.0               hc68bbd7_0    conda-forge
greenlet                  3.0.3           py311hb755f60_0    conda-forge
grpcio                    1.62.2          py311ha6695c7_0    conda-forge
gtk2                      2.24.33              h280cfa0_4    conda-forge
gts                       0.7.6                h977cf35_4    conda-forge
h2                        4.1.0              pyhd8ed1ab_0    conda-forge
h5py                      3.11.0          nompi_py311h439e445_102    conda-forge
harfbuzz                  8.5.0                hfac3d4d_0    conda-forge
hdf5                      1.14.3          nompi_hdf9ad27_105    conda-forge
hpack                     4.0.0              pyh9f0ad1d_0    conda-forge
hydra-colorlog            1.2.0                    pypi_0    pypi
hydra-core                1.3.2                    pypi_0    pypi
hyperframe                6.0.1              pyhd8ed1ab_0    conda-forge
icu                       73.2                 h59595ed_0    conda-forge
idna                      3.7                pyhd8ed1ab_0    conda-forge
imagecodecs               2024.6.1        py311hb8791aa_2    conda-forge
imageio                   2.34.2             pyh12aca89_0    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        8.0.0              pyha770c72_0    conda-forge
importlib_metadata        8.0.0                hd8ed1ab_0    conda-forge
importlib_resources       6.4.0              pyhd8ed1ab_0    conda-forge
iniconfig                 2.0.0              pyhd8ed1ab_0    conda-forge
isodate                   0.6.1              pyhd8ed1ab_0    conda-forge
itsdangerous              2.2.0              pyhd8ed1ab_0    conda-forge
jax                       0.4.27             pyhd8ed1ab_0    conda-forge
jaxlib                    0.4.25          cpu_py311hce36131_0    conda-forge
jinja2                    3.1.4              pyhd8ed1ab_0    conda-forge
joblib                    1.4.2              pyhd8ed1ab_0    conda-forge
jsonschema                4.22.0             pyhd8ed1ab_0    conda-forge
jsonschema-specifications 2023.12.1          pyhd8ed1ab_0    conda-forge
jxrlib                    1.1                  hd590300_3    conda-forge
keras                     3.4.1                    pypi_0    pypi
keyutils                  1.6.1                h166bdaf_0    conda-forge
kiwisolver                1.4.5           py311h9547e67_1    conda-forge
krb5                      1.21.3               h659f571_0    conda-forge
lark-parser               0.12.0             pyhd8ed1ab_0    conda-forge
lazy_loader               0.4                pyhd8ed1ab_0    conda-forge
lcms2                     2.16                 hb7c19ff_0    conda-forge
ld_impl_linux-64          2.40                 hf3520f5_7    conda-forge
lerc                      4.0.0                h27087fc_0    conda-forge
libabseil                 20240116.2      cxx17_h59595ed_0    conda-forge
libaec                    1.1.3                h59595ed_0    conda-forge
libarchive                3.7.4                hfca40fe_0    conda-forge
libarrow                  16.1.0          h4a673ee_10_cpu    conda-forge
libarrow-acero            16.1.0          hac33072_10_cpu    conda-forge
libarrow-dataset          16.1.0          hac33072_10_cpu    conda-forge
libarrow-substrait        16.1.0          h7e0c224_10_cpu    conda-forge
libasprintf               0.22.5               h661eb56_2    conda-forge
libasprintf-devel         0.22.5               h661eb56_2    conda-forge
libavif16                 1.0.4                hd2f8ffe_4    conda-forge
libblas                   3.9.0           22_linux64_openblas    conda-forge
libboost                  1.84.0               hba137d9_3    conda-forge
libboost-python           1.84.0          py311h5510f57_3    conda-forge
libbrotlicommon           1.1.0                hd590300_1    conda-forge
libbrotlidec              1.1.0                hd590300_1    conda-forge
libbrotlienc              1.1.0                hd590300_1    conda-forge
libcblas                  3.9.0           22_linux64_openblas    conda-forge
libclang                  18.1.1                   pypi_0    pypi
libcrc32c                 1.1.2                h9c3ff4c_0    conda-forge
libcurl                   8.8.0                hca28451_1    conda-forge
libdeflate                1.20                 hd590300_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 hd590300_2    conda-forge
libevent                  2.1.12               hf998b51_1    conda-forge
libexpat                  2.6.2                h59595ed_0    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 14.1.0               h77fa898_0    conda-forge
libgd                     2.3.3                h119a65a_9    conda-forge
libgettextpo              0.22.5               h59595ed_2    conda-forge
libgettextpo-devel        0.22.5               h59595ed_2    conda-forge
libgfortran-ng            14.1.0               h69a702a_0    conda-forge
libgfortran5              14.1.0               hc5f4f2c_0    conda-forge
libglib                   2.80.2               h8a4344b_1    conda-forge
libgoogle-cloud           2.25.0               h2736e30_0    conda-forge
libgoogle-cloud-storage   2.25.0               h3d9a0c8_0    conda-forge
libgrpc                   1.62.2               h15f2491_0    conda-forge
libhwloc                  2.10.0          default_h5622ce7_1001    conda-forge
libhwy                    1.1.0                h00ab1b0_0    conda-forge
libiconv                  1.17                 hd590300_2    conda-forge
libidn2                   2.3.7                hd590300_0    conda-forge
libjpeg-turbo             3.0.0                hd590300_1    conda-forge
libjxl                    0.10.2               hcae5a98_0    conda-forge
liblapack                 3.9.0           22_linux64_openblas    conda-forge
libllvm14                 14.0.6               hcd5def8_4    conda-forge
libmicrohttpd             1.0.1                h97afed2_0    conda-forge
libnghttp2                1.58.0               h47da74e_1    conda-forge
libnsl                    2.0.1                hd590300_0    conda-forge
libopenblas               0.3.27          pthreads_h413a1c8_0    conda-forge
libparquet                16.1.0          h6a7eafb_10_cpu    conda-forge
libpng                    1.6.43               h2797004_0    conda-forge
libprotobuf               4.25.3               h08a7969_0    conda-forge
libre2-11                 2023.09.01           h5a48ba9_2    conda-forge
librsvg                   2.58.1               hadf69e7_0    conda-forge
libsqlite                 3.46.0               hde9e2c9_0    conda-forge
libssh2                   1.11.0               h0841786_0    conda-forge
libstdcxx-ng              14.1.0               hc0a3c3a_0    conda-forge
libtasn1                  4.19.0               h166bdaf_0    conda-forge
libthrift                 0.19.0               hb90f79a_1    conda-forge
libtiff                   4.6.0                h1dd3fc0_3    conda-forge
libtorch                  2.3.1           cpu_mkl_h0bb0d08_100    conda-forge
libunistring              0.9.10               h7f98852_0    conda-forge
libunwind                 1.6.2                h9c3ff4c_0    conda-forge
libutf8proc               2.8.0                h166bdaf_0    conda-forge
libuuid                   2.38.1               h0b41bf4_0    conda-forge
libuv                     1.48.0               hd590300_0    conda-forge
libwebp                   1.4.0                h2c329e2_0    conda-forge
libwebp-base              1.4.0                hd590300_0    conda-forge
libxcb                    1.16                 hd590300_0    conda-forge
libxcrypt                 4.4.36               hd590300_1    conda-forge
libxml2                   2.12.7               hc051c1a_1    conda-forge
libzlib                   1.3.1                h4ab18f5_1    conda-forge
libzopfli                 1.0.3                h9c3ff4c_0    conda-forge
lightning                 2.3.1              pyhd8ed1ab_0    conda-forge
lightning-utilities       0.11.3.post0       pyhd8ed1ab_0    conda-forge
llvm-openmp               18.1.8               hf5423f3_0    conda-forge
llvmlite                  0.43.0          py311hbde99c3_0    conda-forge
loguru                    0.7.2           py311h38be061_1    conda-forge
looseversion              1.3.0                    pypi_0    pypi
lz4-c                     1.9.4                hcb278e6_0    conda-forge
lzo                       2.10              hd590300_1001    conda-forge
markdown                  3.6                pyhd8ed1ab_0    conda-forge
markdown-it-py            3.0.0              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.5           py311h459d7ec_0    conda-forge
matplotlib-base           3.8.4           py311ha4ca890_2    conda-forge
matscipy                  1.0.0                    pypi_0    pypi
mdurl                     0.1.2              pyhd8ed1ab_0    conda-forge
memray                    1.13.2          py311h9822ae1_0    conda-forge
mkl                       2023.2.0         h84fe81f_50496    conda-forge
ml-dtypes                 0.3.2                    pypi_0    pypi
mpc                       1.3.1                hfe3b2da_0    conda-forge
mpfr                      4.2.1                h9458935_1    conda-forge
mpmath                    1.3.0              pyhd8ed1ab_0    conda-forge
msgpack-python            1.0.8           py311h52f7536_0    conda-forge
multidict                 6.0.5           py311h459d7ec_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
namex                     0.0.8                    pypi_0    pypi
ncurses                   6.5                  h59595ed_0    conda-forge
nest-asyncio              1.6.0                    pypi_0    pypi
nettle                    3.9.1                h7ab15ed_0    conda-forge
networkx                  3.3                pyhd8ed1ab_1    conda-forge
nnpops                    0.6             cpu_py311h74bf6ba_10    conda-forge
nodejs                    20.12.2              hb753e55_0    conda-forge
numba                     0.60.0          py311h4bc866e_0    conda-forge
numpy                     1.26.4          py311h64a7726_0    conda-forge
omegaconf                 2.3.0                    pypi_0    pypi
opencensus                0.11.3             pyhd8ed1ab_0    conda-forge
opencensus-context        0.1.3           py311h38be061_2    conda-forge
openff-units              0.1.8              pyh1a96a4e_0    conda-forge
openff-utilities          0.1.12             pyhd8ed1ab_0    conda-forge
openjpeg                  2.5.2                h488ebb8_0    conda-forge
openssl                   3.3.1                h4ab18f5_1    conda-forge
opt-einsum                3.3.0                hd8ed1ab_2    conda-forge
opt_einsum                3.3.0              pyhc1e730c_2    conda-forge
optax                     0.2.2                    pypi_0    pypi
optree                    0.11.0                   pypi_0    pypi
orbax-checkpoint          0.5.20                   pypi_0    pypi
orc                       2.0.1                h17fec99_1    conda-forge
p11-kit                   0.24.1               hc5aa10d_0    conda-forge
packaging                 24.1               pyhd8ed1ab_0    conda-forge
pandas                    2.1.4           py311h320fe9a_0    conda-forge
pango                     1.54.0               h84a9a3c_0    conda-forge
patsy                     0.5.6              pyhd8ed1ab_0    conda-forge
pcre2                     10.44                h0f59acf_0    conda-forge
pillow                    10.3.0          py311h82a398c_1    conda-forge
pint                      0.24.1             pyhd8ed1ab_1    conda-forge
pip                       24.0               pyhd8ed1ab_0    conda-forge
pixman                    0.43.2               h59595ed_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge
platformdirs              3.11.0             pyhd8ed1ab_0    conda-forge
pluggy                    1.5.0              pyhd8ed1ab_0    conda-forge
pretty_errors             1.2.25             pyhd8ed1ab_0    conda-forge
prometheus_client         0.20.0             pyhd8ed1ab_0    conda-forge
proto-plus                1.23.0             pyhd8ed1ab_0    conda-forge
protobuf                  4.25.3          py311h7b78aeb_0    conda-forge
psutil                    6.0.0           py311h331c9d8_0    conda-forge
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
py                        1.11.0             pyh6c4a22f_0    conda-forge
py-spy                    0.3.14               h87a5ac0_0    conda-forge
pyarrow                   16.1.0          py311hbd00459_4    conda-forge
pyarrow-core              16.1.0          py311h8c3dac4_4_cpu    conda-forge
pyasn1                    0.6.0              pyhd8ed1ab_0    conda-forge
pyasn1-modules            0.4.0              pyhd8ed1ab_0    conda-forge
pycairo                   1.26.1          py311h64ab44a_0    conda-forge
pycalverter               1.6.1                      py_0    conda-forge
pycparser                 2.22               pyhd8ed1ab_0    conda-forge
pydantic                  1.10.16         py311h331c9d8_0    conda-forge
pygments                  2.18.0             pyhd8ed1ab_0    conda-forge
pyjwt                     2.8.0              pyhd8ed1ab_1    conda-forge
pynndescent               0.5.13             pyhff2d567_0    conda-forge
pyopenssl                 24.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.1.2              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pytest                    8.2.2              pyhd8ed1ab_0    conda-forge
pytest-cov                5.0.0              pyhd8ed1ab_0    conda-forge
python                    3.11.9          hb806964_0_cpython    conda-forge
python-dateutil           2.9.0              pyhd8ed1ab_0    conda-forge
python-graphviz           0.20.3                   pypi_0    pypi
python-tzdata             2024.1             pyhd8ed1ab_0    conda-forge
python-wget               3.2                        py_0    conda-forge
python_abi                3.11                    4_cp311    conda-forge
pytorch                   2.3.1           cpu_mkl_py311hcb16b95_100    conda-forge
pytorch-lightning         2.3.0              pyhd8ed1ab_0    conda-forge
pytorch2jax               0.1.0                    pypi_0    pypi
pytorch_geometric         2.5.3              pyhd8ed1ab_0    conda-forge
pytorch_scatter           2.1.2           cpu_py311h10f7524_6    conda-forge
pytz                      2024.1             pyhd8ed1ab_0    conda-forge
pyu2f                     0.1.5              pyhd8ed1ab_0    conda-forge
pywavelets                1.6.0           py311h18e1886_0    conda-forge
pyyaml                    6.0.1           py311h459d7ec_1    conda-forge
qcelemental               0.25.1             pyhd8ed1ab_2    conda-forge
qcportal                  0.55               pyhd8ed1ab_0    conda-forge
rav1e                     0.6.6                he8a937b_2    conda-forge
ray-core                  2.31.0          py311h0847d76_1    conda-forge
ray-data                  2.31.0          py311h38be061_1    conda-forge
ray-default               2.31.0          py311h48098de_1    conda-forge
ray-train                 2.31.0          py311h38be061_1    conda-forge
ray-tune                  2.31.0          py311h38be061_1    conda-forge
rdflib                    7.0.0              pyhd8ed1ab_0    conda-forge
rdkit                     2024.03.3       py311h1d78c4b_0    conda-forge
re2                       2023.09.01           h7f4b329_2    conda-forge
readline                  8.2                  h8228510_1    conda-forge
referencing               0.35.1             pyhd8ed1ab_0    conda-forge
reportlab                 4.2.2           py311h331c9d8_0    conda-forge
requests                  2.32.3             pyhd8ed1ab_0    conda-forge
retry                     0.9.2                      py_0    conda-forge
rich                      13.7.1             pyhd8ed1ab_0    conda-forge
rlpycairo                 0.2.0              pyhd8ed1ab_0    conda-forge
rpds-py                   0.18.1          py311h5ecf98a_0    conda-forge
rsa                       4.9                pyhd8ed1ab_0    conda-forge
s2n                       1.4.16               he19d79f_0    conda-forge
sake-gnn                  0.post152                pypi_0    pypi
schnetpack                2.0.4                    pypi_0    pypi
scikit-image              0.24.0          py311h14de704_1    conda-forge
scikit-learn              1.5.0           py311he08f58d_1    conda-forge
scipy                     1.14.0          py311h517d4fd_0    conda-forge
setproctitle              1.3.3           py311h459d7ec_0    conda-forge
setuptools                70.1.1             pyhd8ed1ab_0    conda-forge
setuptools-scm            8.1.0              pyhd8ed1ab_0    conda-forge
setuptools_scm            8.1.0                hd8ed1ab_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sleef                     3.5.1                h9b69904_2    conda-forge
smart_open                7.0.4              pyhd8ed1ab_0    conda-forge
snappy                    1.2.0                hdb0a2a9_1    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sphinx                    7.3.7              pyhd8ed1ab_0    conda-forge
sphinx_rtd_theme          2.0.0              pyha770c72_0    conda-forge
sphinxcontrib-applehelp   1.0.8              pyhd8ed1ab_0    conda-forge
sphinxcontrib-devhelp     1.0.6              pyhd8ed1ab_0    conda-forge
sphinxcontrib-htmlhelp    2.0.5              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jquery      4.1                pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1              pyhd8ed1ab_0    conda-forge
sphinxcontrib-qthelp      1.0.7              pyhd8ed1ab_0    conda-forge
sphinxcontrib-serializinghtml 1.1.10             pyhd8ed1ab_0    conda-forge
sqlalchemy                2.0.31          py311h331c9d8_0    conda-forge
sqlite                    3.46.0               h6d4b2fc_0    conda-forge
sqlitedict                2.1.0              pyhd8ed1ab_0    conda-forge
statsmodels               0.14.2          py311h18e1886_0    conda-forge
svt-av1                   2.1.0                hac33072_0    conda-forge
sympy                     1.12.1          pypyh2585a3b_103    conda-forge
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tbb                       2021.12.0            h297d8ca_1    conda-forge
tensorboard               2.16.2                   pypi_0    pypi
tensorboard-data-server   0.7.0           py311h63ff55d_1    conda-forge
tensorboardx              2.6.2.2            pyhd8ed1ab_0    conda-forge
tensorflow                2.16.2                   pypi_0    pypi
tensorflow-io-gcs-filesystem 0.37.0                   pypi_0    pypi
tensorstore               0.1.63                   pypi_0    pypi
termcolor                 2.4.0                    pypi_0    pypi
textual                   0.71.0             pyhd8ed1ab_0    conda-forge
threadpoolctl             3.5.0              pyhc1e730c_0    conda-forge
tifffile                  2024.6.18          pyhd8ed1ab_0    conda-forge
tk                        8.6.13          noxft_h4845f30_101    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.1                   pypi_0    pypi
torch-ema                 0.3                      pypi_0    pypi
torchani                  2.2.4           cpu_py311hae4df2b_8    conda-forge
torchmd-net               2.2.0           cpu_py311h430139b_3    conda-forge
torchmetrics              1.0.1                    pypi_0    pypi
torchvision               0.18.1          cpu_py311hf0a5325_1    conda-forge
torchviz                  0.0.2                    pypi_0    pypi
tqdm                      4.66.4             pyhd8ed1ab_0    conda-forge
trimesh                   4.4.1              pyhd8ed1ab_0    conda-forge
typing-extensions         4.12.2               hd8ed1ab_0    conda-forge
typing_extensions         4.12.2             pyha770c72_0    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
urllib3                   2.2.2              pyhd8ed1ab_1    conda-forge
versioneer                0.29               pyhd8ed1ab_0    conda-forge
virtualenv                20.21.0            pyhd8ed1ab_0    conda-forge
werkzeug                  3.0.3              pyhd8ed1ab_0    conda-forge
wheel                     0.43.0             pyhd8ed1ab_1    conda-forge
wrapt                     1.16.0          py311h459d7ec_0    conda-forge
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.1.1                hd590300_0    conda-forge
xorg-libsm                1.2.4                h7391055_0    conda-forge
xorg-libx11               1.8.9                hb711507_1    conda-forge
xorg-libxau               1.0.11               hd590300_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-libxext              1.3.4                h0b41bf4_2    conda-forge
xorg-libxrender           0.9.11               hd590300_0    conda-forge
xorg-renderproto          0.11.1            h7f98852_1002    conda-forge
xorg-xextproto            7.3.0             h0b41bf4_1003    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.6                h166bdaf_0    conda-forge
yaml                      0.2.5                h7f98852_2    conda-forge
yarl                      1.9.4           py311h459d7ec_0    conda-forge
zfp                       1.0.1                hac33072_1    conda-forge
zipp                      3.19.2             pyhd8ed1ab_0    conda-forge
zlib                      1.3.1                h4ab18f5_1    conda-forge
zlib-ng                   2.2.0                hac33072_0    conda-forge
zstandard                 0.22.0          py311hb6f056b_1    conda-forge
zstd                      1.5.6                ha6fb4c9_0    conda-forge
```

**Comment by MarshallYan** [comment]
#167 Solved by specifying 'pint=0.23'.
